docs/versioned_docs/version-v8.5.x/03-light-clients/02-localhost/03-client-state.md

```diff
@@ -33,7 +33,7 @@ It is possible to disable the localhost client by removing the `09-localhost` en
 
 The latest height is updated periodically through the ABCI [`BeginBlock`](https://docs.cosmos.network/v0.47/building-modules/beginblock-endblock) interface of the 02-client submodule in core IBC.
 
-[See `BeginBlocker` in abci.go.](https://github.com/cosmos/ibc-go/blob/09-localhost/modules/core/02-client/abci.go#L12)
+[See `BeginBlocker` in abci.go.](https://github.com/cosmos/ibc-go/blob/main/modules/core/02-client/abci.go#L12)
```

Comment:
I don't think this link is quite correct either. It is not dead, but it is also not going to where it should :)

Reply:
Agree with that, I just modified that to L28. Could you please check it?

'The latest height is updated periodically via the ABCI BeginBlock interface on the 02 client submodule of the core IBC.

See BeginBlocker at abci.go.'

I believe this corresponds to 

https://github.com/cosmos/ibc-go/blob/e10bbbc47fb70e138a3369902d18586b3da95800/modules/core/02-client/abci.go#L28